onnxmltools/convert/coreml/operator_converters/TensorToLabel.py

```diff
@@ -66,7 +66,11 @@ def convert_tensor_to_label(scope, operator, container):
     label_loader_attrs = {'name': label_loader_name}
     label_buffer_name = scope.get_unique_variable_name('ClassLabels')
     label_loader_attrs['value'] = helper.make_tensor(label_buffer_name, label_type, [len(labels)], labels)
-    container.add_node('Constant', [], [label_buffer_name], **label_loader_attrs)
+    if container.target_opset < 9:
```

Comment:
should we create a apply_XXX-like function for it?

Reply:
Makes sense! I'll add apply_constant so we don't have to repeat the logic in multiple places.